addons/bestja_account_deletion/wizards.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+from openerp import models, fields, api
+
+
+class Wizard(models.TransientModel):
+    _name = 'bestja.delete_password_wizard'
+    
+    def _default_users(self):
+        return self.env['res.users'].browse(self.env.context.get('active_ids'))
+
+
+    users = fields.Many2many(
+        'res.users',
+        string="Użytkownicy",
+        required=True,
+        default=_default_users,
+    )
+
+    @api.one
+    def delete_account(self):
+        for user in self.users:
```

Comment:
Since `self.users` is a recordset and  `delete_account()` is (should be) decoreted with `api.one` you should be able to just do

```
self.users.delete_account()
```

without any explicit loops.